api/src/main/resources/workbench.yaml

```diff
@@ -3329,11 +3584,7 @@ definitions:
 
   PrePackagedConceptSetEnum:
     type: string
-    enum:
-      - None
-      - Demographics
-      - Survey
-      - Both
+    enum: &PrePackagedConceptSetEnum [NONE, DEMOGRAPHICS, SURVEY, BOTH]
```

Comment:
This feels like it could potentially be a breaking change, depending on which of the conflicting enums "won out" in our resulting Swagger-codegen world. What gives you confidence that this is the correct resolution, using the all-caps variant?

Reply:
Removing this one caused compiler failures, and removing the other one didn't. That suggests to me that maybe the caps one had been overriding the non-caps one. (hey look! One more reason to consolidate!)